google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -757,6 +760,71 @@ def lock_retention_policy!
           true
         end
 
+        ##
+        # Whether the bucket's file IAM configuration enables Bucket Policy
+        # Only. The default is false. This value can be modified by calling
+        # {Bucket#policy_only=}.
+        #
+        # If true, access checks only use bucket-level IAM policies or above,
+        # all object ACLs within the bucket are no longer evaluated, and
+        # access-control is configured solely through the bucket's IAM policy.
+        # Any requests which attempt to use the ACL API to view or manipulate
+        # ACLs will fail with 400 errors.
+        #
+        # @return [Boolean] Returns `false` if the bucket has no IAM
+        #   configuration or if Bucket Policy Only is not enabled in the IAM
+        #   configuration. Returns `true` if Bucket Policy Only is enabled in
+        #   the IAM configuration.
+        #
+        # @example
+        #   require "google/cloud/storage"
+        #
+        #   storage = Google::Cloud::Storage.new
+        #
+        #   bucket = storage.bucket "my-bucket"
+        #
+        #   bucket.policy_only = true
+        #   bucket.policy_only? # true
```

Comment:
Please expose the field: `iamConfiguration.lockedTime`. A user should be able to check when this field can no longer be reverted.

Reply:
@frankyn @blowmage I exposed `lockedTime` as `Bucket#policy_only_locked_at`. This name complies with the existing naming convention for DateTime fields, such as `Bucket#retention_effective_at`.

Reply:
👍

Reply:
lgtm!

Reply:
One nit: @quartzmo please add `bucket.retention_effective_at?` into the example above to relate the methods.

Reply:
@quartzmo, could you add `puts bucket.policy_only_locked_at?` into this example?

Reply:
Updated.